Networking/Networking/Model/WCPayAccount.swift

```diff
@@ -37,7 +38,7 @@ public struct WCPayAccount: Decodable {
         self.supportedCurrencies = supportedCurrencies
         self.country = country
         /// Hardcoded until support for this property is available in WCPay
-        self.canCollectPayments = true
+        self.isCardPresentEligible = true
```

Comment:
This is what we'd need to change in #4241 , and set the property to the value passed as parameter, instead f hardcoding it to true